notes/fidelity.html

```diff
@@ -71,6 +66,11 @@ <h2 class="ui header">
         <li>
             Call Fidelity Account Key Support at +1 800-544-6666.
         </li>
+        <li>
+            Once logged in, navigate to <b>Profile</b> &gt; <b>Security
+            Center</b> &gt; <b> 2-factor authentication</b>. This process only
+            supports SMS.
```

Comment:
I have an ordinary personal account at Fidelity.com (not set up through an employer). I signed up for 2FA with my phone number. When I log in via an incognito/private browser I get prompted after my password for whether I'd like a text message or automated voice call to receive a security code, so it seems this isn't true.